Tests/AppTests/PackageTests.swift

```diff
@@ -756,11 +756,46 @@ final class PackageTests: AppTestCase {
         }
 
         // MUT
-        let res = p.swiftVersionCompatibility()
+        let res = p.swiftVersionCompatibility().values
 
         // validate
         XCTAssertEqual(res.sorted(), [.v5_2, .v5_3])
     }
+    
+    func test_swiftVersionCompatibility_withPending() throws {
+        // setup
+        let p = try savePackage(on: app.db, "1")
+        let v = try Version(package: p, reference: .tag(.init(1, 2, 3)))
+        try v.save(on: app.db).wait()
+        // re-load repository relationship (required for updateLatestVersions)
+        try p.$repositories.load(on: app.db).wait()
+        // update versions
+        _ = try updateLatestVersions(on: app.db, package: p).wait()
+        // add builds
+        try Build(version: v, platform: .linux, status: .ok, swiftVersion: .init(5, 3, 0))
+            .save(on: app.db)
+            .wait()
+        try Build(version: v, platform: .macosXcodebuild, status: .ok, swiftVersion: .init(5, 2, 2))
+            .save(on: app.db)
+            .wait()
+        try Build(version: v, platform: .ios, status: .pending, swiftVersion: .init(5, 0, 2))
+            .save(on: app.db)
+            .wait()
+        try p.$versions.load(on: app.db).wait()
+        try p.versions.forEach {
+            try $0.$builds.load(on: app.db).wait()
+        }
+
+        // MUT
+        let res = p.swiftVersionCompatibility()
+
+        // validate
+        if case .pending = res {
+            return
+        }
+        
+        XCTFail("Compatibility result should have been set to pending")
```

Comment:
I'd make this

```swift
        // validate
        XCTAssertEqual(res, .pending)
```

via

```swift
    enum CompatibilityResult<Value: Equatable>: Equatable {
```

(It's a bit more constrained than technically necessary but if that's a concern we could also only make it `Equatable` in the test target - but I think we don't need to be concerned about that at this time and it makes the test nicer :)